src/Sulu/Bundle/AdminBundle/Tests/Functional/Metadata/FormMetadata/FormMetadataProviderTest.php

```diff
@@ -50,4 +50,30 @@ public function testGetMetadataFromStructureLoader()
         $this->assertInstanceOf(TypedFormMetadata::class, $typedForm);
         $this->assertCount(2, $typedForm->getForms());
     }
+
+    public function testGetMetadataTagFiltered()
+    {
+        $typedForm = $this->formMetadataProvider->getMetadata('page', 'de', ['tags' => ['test' => 'test-value']]);
+        $this->assertInstanceOf(TypedFormMetadata::class, $typedForm);
+        $this->assertCount(1, $typedForm->getForms());
+        $this->assertEquals(['default'], array_keys($typedForm->getForms()));
+
+        $typedForm = $this->formMetadataProvider->getMetadata('page', 'de', ['tags' => ['test' => 'test-value2']]);
+        $this->assertInstanceOf(TypedFormMetadata::class, $typedForm);
+        $this->assertCount(0, $typedForm->getForms());
+
+        $typedForm = $this->formMetadataProvider->getMetadata('page', 'de', ['tags' => ['test2' => 'test-value2']]);
+        $this->assertInstanceOf(TypedFormMetadata::class, $typedForm);
+        $this->assertCount(0, $typedForm->getForms());
+
+        $typedForm = $this->formMetadataProvider->getMetadata('page', 'de', ['tags' => ['test2' => 'test-value2'], 'tagAttributes' => ['test2' => 'test']]);
```

Comment:
What does `['test2' => 'test']` mean in that case? I don't really understand what is being filtered here by just reading that line.